app/views/mfa_confirmation/show.html.erb

```diff
@@ -2,21 +2,23 @@
 
 <%= image_tag asset_url('user-signup-ial1.svg'), width: 107, height: 119, alt: '', class: 'margin-bottom-4' %>
 
-<%= render PageHeadingComponent.new.with_content(t('titles.mfa_setup.suggest_second_mfa')) %>
+<%= render PageHeadingComponent.new.with_content(@content.heading) %>
 
-<p class='margin-top-1 margin-bottom-4'><%= t('mfa.account_info') %></p>
+<p class='margin-top-1 margin-bottom-4'><%= sanitize @content.info %></p>
```

Comment:
I think we can remove `sanitize` (it's a code smell to me) because the i18n key ends in `_html` and it not... then we are doing something else wrong.
```suggestion
<p class='margin-top-1 margin-bottom-4'><%= @content.info %></p>
```

Reply:
`@content.info.html_safe` works too, is that better in terms of code smell?

Reply:
That means that we're not using i18n to escape properly:

https://github.com/18F/identity-idp/blob/a3bb7c57047c39ff365f87d56f548098e99a6e01/app/presenters/mfa_confirmation_presenter.rb#L17-L22

My guess looking at the source is we need to rename the `mfa.account_info` key to have `_html` at the end so it will auto-escape

more info here: https://guides.rubyonrails.org/i18n.html#using-safe-html-translations

Reply:
As discussed, I pulled `learn more` out into its own link and put it behind some logic on the view to determine whether to show it or not.